### ISSUE TITLE
Unclosed quote error observed when all quotes are properly balanced

### ISSUE THREAD

**Issue by cookch10** [stylelint/stylelint#1150]
When using stylelint `v6.2.2` as follows: 

package.json (simplified):
```json
{
  "scripts": {
    "lint:styles": "stylelint --syntax scss \"src/**/*.scss\""
  }
}
```

My .stylelintrc config (uses stylelint-config-standard `v6.0.0`):
```json
{
  "extends": "stylelint-config-standard",
  "rules": {
    "function-url-quotes": "none",
    "selector-pseudo-element-colon-notation": "single"
  }
}
```

given the following Sass (scss file):
```scss
@import "../../../styles/config";

.some-widget {
    $column: #{"#{&}__column"};

    > .row {
        > #{$column}__section {
            .some__section {
                @include some-breakpoint-mixin("desktop") {
                    display: none;
                }
            }
        }
    }
}
```

```shell
npm run lint:styles
```

The following results in an error:
```shell
 9:49  ×  Unclosed quote   CssSyntaxError
```

This is valid Sass and should not be yielding an unclosed quote error.  Possibly related to #1108 / parser issue in [postcss-scss](https://github.com/postcss/postcss-scss).

**Comment by evilebottnawi** [comment]
Confirm, i think it is bug in postcss-scss, not stylelint

**Comment by cookch10** [comment]
@evilebottnawi, should I open an issue at [postcss-scss](https://github.com/postcss/postcss-scss)?

**Comment by davidtheclark** [comment]
It would help first to narrow down your code in order to identify which particular quotation marks are not being interpreted correctly.

**Comment by cookch10** [comment]
If I change the following:

```scss
$column: #{"#{&}__column"};
```

to

```scss
$column: "column";
```

The Unclosed quote error is not thrown.  So it seems there may be an issue with parsing quotes in interpolated Sass strings / variables.

**Comment by davidtheclark** [comment]
Makes sense. That would be a good issue to open up in postcss-scss.


On Tue, Apr 26, 2016 at 11:49 AM, Christopher Cook <notifications@github.com

**Comment by cookch10** [comment]
Issue created:  https://github.com/postcss/postcss-scss/issues/30

**Comment by davidtheclark** [comment]
@stylelint/core I created type:scss and type:less labels so that we can clearly distinguish (and possibly de-prioritize) bugs that are preprocessor-specific. Does that sound good?

**Comment by jeddy3** [comment]
> Does that sound good?

Looks great. FYI, I coloured them to match each brands most prominent colour :)

**Comment by evilebottnawi** [comment]
Done. Ref: https://github.com/postcss/postcss-scss/issues/30

**Comment by cookch10** [comment]
@evilebottnawi, will you be publishing a new release to include this fix?

**Comment by evilebottnawi** [comment]
@cookch10 sorry, not now, in the near future I think we will release a new version :tada:

**Comment by ntwb** [comment]
@cookch10 No need for a new release, if you run `npm update` that should update the stylelint `postcss-scss` dependency to this new `v0.1.8` version of `postcss-scss` 😄

**Comment by cookch10** [comment]
@ntwb, thanks for that! :thumbsup:
